src/i18n/es.js

```diff
@@ -503,6 +503,7 @@ eliminación, seguridad, transferencias transfronterizas y otros temas.
     value_false: 'Rechazado',
     value_none: 'Sin respuesta',
     value_true: 'Aceptado',
+    error_connection_problem: `¡Ouch! Hubo un problema de conexión. <button>Reintenta aquí</button>`,
```

Comment:
We have a text for this already [here ](https://github.com/FromDoppler/doppler-webapp/blob/master/src/i18n/es.js#L577) Perhaps you could reuse it.

Reply:
In this case it is not useful to me because the text 'Retry here' is used as a button.

Reply:
You could leave the retry outside scope for this PR. We don´t have any retry elsewhere.

Reply:
> You could leave the retry outside scope for this PR. We don´t have any retry elsewhere.

Great idea! in that way we could avoid the bug mentioned before: https://github.com/FromDoppler/doppler-webapp/pull/1333#discussion_r612376679

Reply:
Sounds good to me, so we reduce the scope of the PR.